index.html

```diff
@@ -0,0 +1,292 @@
+            </p>
+          </div>
+        </div>
+        <div class="investors-list">
+          <img
+            src="./images/phone-view-icons/investors_02.png"
+            alt="investors-icon2"
+            class="investors-img"
+          />
+          <div class="investors-content">
+            <h4>Kilnam Chon</h4>
+            <h5>Chef in Hotel-palace</h5>
+            <hr class="investors-line" />
+            <p class="investors-description">
+              Discover achu and yellow soupe, explaine how it can cure some
+              certain illness.
+            </p>
+          </div>
+        </div>
+        <div class="investors-list">
+          <img
+            src="./images/phone-view-icons/investors_03.png"
+            alt="investors-icon3"
+            class="investors-img"
+          />
+          <div class="investors-content">
+            <h4>sohyeong Noh</h4>
+            <h5>Chef in Hotel-china-town</h5>
+            <hr class="investors-line" />
+            <p class="investors-description">
+              coming directly from HongKong to give her advice and encourage of
+              this new restaurant.
+            </p>
+          </div>
+        </div>
+        <div class="investors-list">
+          <img
+            src="./images/phone-view-icons/investors_04.png"
+            alt="investors-icon4"
+            class="investors-img"
+          />
+          <div class="investors-content">
+            <h4>Julia Reda</h4>
+            <h5>Representing Chef in hotel-picasso in France</h5>
+            <hr class="investors-line" />
+            <p class="investors-description">
+              One of the best chef crown by the president of France itself.
+            </p>
+          </div>
+        </div>
+        <div class="investors-list">
+          <img
+            src="./images/phone-view-icons/investors_05.png"
+            alt="investors-icon5"
+            class="investors-img"
+          />
+          <div class="investors-content">
+            <h4>Lila Treticov</h4>
+            <h5>Secretary of chef in russia of the department of brevrage</h5>
+            <hr class="investors-line" />
+            <p class="investors-description">
+              Contribution in the exporting and importing of goods and services.
+            </p>
+          </div>
+        </div>
+        <div class="investors-list">
+          <img
+            src="./images/phone-view-icons/investors_06.png"
+            alt="investors-icon6"
+            class="investors-img"
+          />
+          <div class="investors-content">
+            <h4>Ryan Merkley</h4>
+            <h5>President in the chef sector in hotel-Torento</h5>
+            <hr class="investors-line" />
+            <p class="investors-description">
+              Incharge of investigation of how food have to be cooked in a
+              restaurant.
+            </p>
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
- Kindly, This section has to be generated dynamically with JavaScript as mentioned in the project requirement. 👍 

the list of investors used to generate the HTML is assigned to a variable as an array or object.
```
{
  name: "Julia Leda", 
  role: "President of Young pirates of Europe", 
  description: "European ingetration, political democracy, and participation of ...", 
  photo: "image_url_here"
}
```
then you can iterate through the investor list on the window load and creates HTML elements with the data from each object. For example:
```
window.onload = () => {
  speakerList.map(speaker => {
    // code to build the new HTML element, it can use `createElement()` or just template strings(``).
    // it could attach the new element to a parent inside within the loop
    // or accumulate the generate elements in a variable and attach them all after it
  })
}
```